extra/test/mock_os.h

```diff
@@ -69,6 +69,8 @@ struct MockOs
 
     static bool IsInIsr(void) { return false; }
 
+    static int64_t GetTimeMs(void) { return m_now_ms; }
```

Comment:
Nit: `m_now_ms ` need to be signed?

Reply:
I chose a signed integer because people tend to take differences between times (and signed/unsigned comparisons can be tricky), and the notion of a negative time (or difference between times) is useful. Less important, this matches the type for time in our internal repo.